### PR TITLE
fix modulesUrl old reference

### PR DESCRIPTION
**Pull request by geo1211**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@geo1211__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@geo1211__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: 101-vm-simple-linux/flash-vm-linux.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```
Comment (singhkay) [inline 101-vm-simple-linux/flash-vm-linux.json]: @geo1211 what is the purpose of this file? This seems too simple plus there is already a template here

Comment (geo1211) [inline 101-vm-simple-linux/flash-vm-linux.json]: sorry this file was a mistake.  pulling it out now

**Comment by singhkay** [comment]
@geo1211 lgtm, thanks for fixing. Merged

**Pull request merged**